### PR TITLE
841 add support for secrets for jdbc connections

### PR DESCRIPTION
**Pull request by itsabhisharma23**
fix:  Added secret capabilities for all python jdbc templates.

### CODE REVIEW

File: python/dataproc_templates/jdbc/README.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ jdbc:sqlserver://<hostname>:<port>;databaseName=<dbname>;user=<username>;passwor
 ```
 jdbc:oracle:thin:@//<hostname>:<port>/<dbservice>?user=<username>&password=<password>
 ```
+* Note:
```
Comment (surjits254) [inline python/dataproc_templates/jdbc/README.md:59]: can we make note bold?

File: python/dataproc_templates/jdbc/README.md [hunk 2]
```diff
@@ -95,10 +97,12 @@ Template for reading data from JDBC table and writing them to a JDBC table. It s
 
 ## Arguments
 
-* `jdbctojdbc.input.url`: JDBC input URL
+* `jdbctojdbc.input.url`: JDBC input URL 
+* `jdbctojdbc.input.url.secret`: JDBC input URL secret. Pass the secret name as created in Cloud Secret Manager.
```
Comment (surjits254) [inline python/dataproc_templates/jdbc/README.md:101]: can we specify only one of 2 params are required all through the readme?

**Review by surjits254 — COMMENTED** [review]
minor changes in readme

File: python/dataproc_templates/jdbc/README.md (hunk 1 above)
Comment (itsabhisharma23) [reply python/dataproc_templates/jdbc/README.md]: Done

**Review by vanshaj-bhatia — APPROVED** [review]
LGTM!

**Pull request merged**